### ISSUE TITLE
Error while deserailzation

### ISSUE THREAD

**Issue by ravgupta-ms** [geoffodonnell/dotnet-tinyman-sdk#11]
Looks like algorand api are no longer returning "creator" field for assets. This fields is required, so all tinyman api like GetBalance, FetchAssets are failing.

**Comment by cryptoDotnet** [comment]
I have encountered the same problem
```
{Newtonsoft.Json.JsonSerializationException: Required property 'creator' not found in JSON. Path 'assets[0]', line 1, position 1340.
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.EndProcessProperty(Object newObject, JsonReader reader, JsonObjectContract contract, Int32 initialDepth, JsonProperty property, PropertyPresence presence, Boolean setDefaultValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
etc
```

**Comment by geoffodonnell** [comment]
This issue exists in Algorand .NET SDK by RileyGe. @FrankSzendzielarz is actively maintaining a fork of the original, which has addressed this issue. As such, the Tinyman SDK now references the Algorand2 fork.

Addressed in Tinyman 0.3.0.1.

**Comment by FrankSzendzielarz** [comment]
Please also let me know directly on the Algorand Discord .NET SDK channel if there are any other SDK issues. Recently there were core updates and the .NET community tooling will be updated to incorporate those in the next few days. This month (April) a project will begin to essentially make the .NET SDK a core Algorand SDK.
